locales/locales/en.json

```diff
@@ -1629,6 +1629,27 @@
 	"accountSwitcherReloadOtherTabsDesc": {
 		"message": "After switching accounts, automatically reload other tabs."
 	},
+	"accountSwitcherAutoSwitchOnPostTitle": {
+		"message": "Automatically Switch To Post OP"
+	},
+	"accountSwitcherAutoSwitchOnPostDesc": {
+		"message": "Automatically switch to alternative account if it's the OP of the post you are on."
```

Comment:
How about ...

"Automatically switch to account in Account Switcher if that account submitted the current post."